sc-memory/tests/sc-memory/common/sc-memory.cpp

```diff
@@ -79,3 +98,66 @@ TEST_F(ScMemoryTest, ResolveNodeWithRussianIdtf)
   EXPECT_TRUE(m_ctx->HelperSetSystemIdtf(englishIdtf, englishNode));
   EXPECT_EQ(m_ctx->HelperFindBySystemIdtf(englishIdtf), englishNode);
 }
+
+TEST_F(ScMemoryTestWithResultStruct, TestResultStruct)
```

Comment:
This test should also check that link with system idtf of keynode, edges to this links, also was added to result.

test_keynode
 => [test_keynode]
  ^
  |
 nrel_system_identificator